### DISCUSSION TITLE
Issue with User Identification in AdGuard Home Query Log

### DISCUSSION THREAD

**Discussion by yosefhalp** [AdguardTeam/AdGuardHome#7236] [Q&A]
Hello everyone,

I'm currently using the AdGuard Home system and have it set up as a DHCP server, even though my Xiaomi AX5400 router also has DHCP capabilities. To avoid conflicts, I've disabled the DHCP setting on the router and rely solely on AdGuard Home for DHCP management. I've configured a fixed DNS on the router, which is linked to my AdGuard Home setup, and everything seems to be working perfectly. My AdGuard Home is running on a Linux server that I built on a DELL OptiPlex 7040 computer. The filtering works as expected, and overall, the system is performing well.

However, I've encountered an issue that I can't seem to resolve. When I check the query log in AdGuard Home, I'm only seeing the router's IP address (192.168.31.1) instead of the individual IP addresses of the devices connected to the network. This makes it difficult to identify which device is making specific DNS queries.

To troubleshoot, I also tried using only the router's DHCP server and disabled AdGuard Home's DHCP, but I encountered the same problem. The query log continues to show only the router's IP address rather than the IP addresses of the individual devices.

Here’s a summary of my setup:
- **Router:** Xiaomi AX5400
- **DHCP Server:** AdGuard Home (router’s DHCP disabled)
- **DNS Configuration:** Fixed DNS set on the router, pointing to AdGuard Home
- **Server:** Running on a DELL OptiPlex 7040 with Linux

Despite everything working well in terms of DNS filtering and network performance, the query log only displays the router's IP address instead of the IP addresses of individual devices. 

Has anyone else experienced this issue? Is there a specific configuration that I might be missing to ensure that AdGuard Home logs the actual IP addresses of the devices rather than just the router's address? Any advice or suggestions would be greatly appreciated!

Thanks in advance for your help!
![image](https://github.com/user-attachments/assets/4d33aa6f-f338-4772-bd35-0119f26910fa)

**Comment by cameronj86** [comment]
I believe #7064 is related and similarly unresolved
